packages/ui/src/components/molecules/ProductTitle/styles.scss

```diff
@@ -0,0 +1,55 @@
+[data-fs-product-title] {
+  // --------------------------------------------------------
+  // Design Tokens for Product Title
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-product-title-text-size            : var(--fs-text-size-title-product);
+  --fs-product-title-text-weight          : var(--fs-text-weight-regular);
+  --fs-product-title-line-height          : 1.12;
+  --fs-product-title-column-gap           : var(--fs-spacing-2);
+  --fs-product-title-row-gap              : var(--fs-spacing-3);
+
+  --fs-product-title-addendum-margin-top  : var(--fs-spacing-3);
```

Comment:
Suggestion: add a `row-gap` on `[data-fs-product-title]` and use `--fs-product-title-row-gap` instead of `--fs-product-title-addendum-margin-top` 🙃